main.py

```diff
@@ -0,0 +1,27 @@
+from fastapi import FastAPI
+from pydantic import BaseModel
+from fastapi import FastAPI, HTTPException
```

Comment:
The `FastAPI` import is duplicated. You can remove one of the `from fastapi import FastAPI` lines to clean up the imports.